src/tagrec.py

```diff
@@ -147,18 +148,30 @@ def make_decision(self, direction):
         return decision
 
 
-    def detect(self):
+    def detect(self, img_src=None):
         """
         Detect an ARTag.
 
+        :param img_src: Used to bypass the camera feed and track ARTags from
+                        images instead. Used primarily for testing purposes.
+                        Disabled by default.
+        :type img_src: str
+
         :return: Dictionary containing the x distance, z distance, direction,
                  decision, and yaw if an ARTag is detected.
         :rtype: float, float, float, int, float
 
         :return: None if an camera does not recognize an ARTag.
         :rtype: None
         """
-        self._ret, self._frame = self._cap.read()
+        if not img_src:
+            self._ret, self._frame = self._cap.read()
+        else:
+            file_exists = os.path.isfile(img_src)
+            if file_exists:
+                self._frame = cv2.imread(img_src)
+            else:
+                raise FileNotFoundError('File does not exist')
```

Comment:
If we're raising an exception, it needs to be in the docstring.

`:raises FileNotFoundError:`